index.d.ts

```diff
@@ -29,6 +29,7 @@ declare namespace GSF {
         info(): Promise<ServiceInfo>;
         task(taskName: string): Task;
         tasks(): Promise<Array<Task>>;
+        taskInfoList(): Promise<Array<TaskInfo>>;
```

Comment:
the other function names are all nouns though, so not sure if "starts with a verb" is the precedent